lib/laa_crime_schemas/types/types.rb

```diff
@@ -287,6 +287,8 @@ module Types
                                          no_check_required
                                          checker_unavailable
                                          confirmed])
+
+    BusinessType = String.enum(*%w[self_employed partnership director_or_shareholder])
```

Comment:
Any thoughts on naming this BusinessInvolvementType?

Reply:
Hmm, I've just seen Tim's PR- didn't realise this was the page name/in the designs. I'm not a fan. But that's fine

Reply:
Happy to change to business involvement type - i see how that's a lot more accurate